src/TestFramework/PhpUnit/Adapter/PhpUnitAdapter.php

```diff
@@ -52,6 +53,8 @@ class PhpUnitAdapter extends AbstractTestFrameworkAdapter implements IgnoresAddi
 {
     public const COVERAGE_DIR = 'coverage-xml';
 
+    private const MINIMUM_SUPPORTED_VERSION = '6.0.0';
```

Comment:
version up for debate: I would argue it may be time to push for a higher version, e.g. one that at least supports pcov

Reply:
`pcov` support is added in `8.0`. However, `8.0` has a very annoying BC break - removed [`assertArraySubset`](https://github.com/sebastianbergmann/phpunit/issues/3494#issuecomment-464073900) which from what I know prevents some devs from upgrading (there is a separate package for it though).

Personally, I'm ok even with bumping it to `8.0`, but I can't answer to myself why would we do it.

No infection works perfectly with PHPUnit `5.2`, the only issue I found is that `pcov` is not supported.

Reply:
I would like to provide a better experience than waiting for the test framework to fully run to get an error saying that the coverage could not be generated :/

Reply:
Maybe it's just too optimistic of me: since PHPUnit does not support it, I don't think we should either

Reply:
so what version are you suggesting to stick to and why? 6.0.0? 8.0.0?